### ISSUE TITLE
Transaction Rejected - Unable to vote from second account 

### ISSUE THREAD

**Issue by Filipv95** [DXgovernance/DAVI#246]
**Description**
Basically I have created proposal with first account and voted with it, then I've switched to second account and was not able to vote (also tried resetting account on Metamask) 

**Steps to Reproduce**
1. SpinUp Localhost env
2. Open localhost:3000
3. Check if Landing Page is loaded
4. Connect wallet to Metamask
5. Create Proposal with 'Transfer & Mint' action with first account
6. Vote For that Proposal to execute successfully
7. Switch to second account
8. Vote For that Proposal to execute successfully

**Expected behavior**
Should be able to successfully vote with accounts that have joined the Guild and have enough power.

**Screenrecord**
https://user-images.githubusercontent.com/37701286/186777120-88aa246b-62c3-44f6-ac45-b49c5c56950a.mp4

**Device Information**
OS: MacOs
Browser: GoogleChrome, Brave
Device: MacOS Monterey v12.4

**Additional context**
Was able to successfully execute proposal with second account.
Also I've created another proposal with same action and was able to vote from both accounts successfully.

**Comment by rossneilson** [comment]
Can you see what the timings look like here in the proposal history section, could be localhost timings being weird?

**Comment by Filipv95** [comment]
@rossneilson hey, timings were good, but this is what I figured out and need affirmation whether that is expected behavior or not.

Basically, 
1. there is 1 Guild member which created Proposal > votes for proposal sucessfully > second member joins > he is unable to vote for that proposal
2. there are 2 members when Proposal is created > both of them can vote successfully

Tried this with all 3 accounts that have SWPR tokens, works the way I described it above.

So tldr, if a user was not a member when Proposal was created he won't be able to vote regardless of his voting power, is that right?

**Comment by rossneilson** [comment]
Yes this is the exact expected behaviour, good job describing it as such.
This is related to #254 and we should test again afterwards. 

Also going to investigate a bit myself, @Filipv95 is the behaviour you see when you say "second member joins > he is unable to vote for that proposal" that the user cannot see voting buttons or that the vote transaction fails?

**Comment by Filipv95** [comment]
Tnx @rossneilson good to know.

 "second member joins > he is unable to vote for that proposal" - transaction fails

**Comment by Filipv95** [comment]
@rossneilson basically we can close this issue, since the flow was not correct in the first place, right?

**Comment by Filipv95** [comment]
@rossneilson ATM doing some regression testing and I've joined SWPR Guild with all 3 accounts, then created Proposals for '**Set Permission**' and '**Transfer**' and I have noticed that sometimes I'm not able to vote from all accounts on all proposals that I've created (sometimes I can vote from all 3 accounts). 
Do you have any idea why this might be the case?
